core/pom.xml

```diff
@@ -44,12 +44,22 @@
     <dependency>
       <groupId>commons-beanutils</groupId>
       <artifactId>commons-beanutils</artifactId>
-      <version>1.7.0</version>
+      <version>1.8.0</version>
```

Comment:
Are these aligned with `jenkins-core`?

Reply:
Jenkins core is at 1.8.3. Can bump it even more